google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -167,15 +169,19 @@ public Blob create(
   }
 
   @Override
-  @Deprecated
-  public Blob create(BlobInfo blobInfo, InputStream content, BlobWriteOption... options) {
-    Tuple<BlobInfo, BlobTargetOption[]> targetOptions = BlobTargetOption.convert(blobInfo, options);
-    StorageObject blobPb = targetOptions.x().toPb();
-    Map<StorageRpc.Option, ?> optionsMap = optionMap(targetOptions.x(), targetOptions.y());
-    InputStream inputStreamParam =
-        firstNonNull(content, new ByteArrayInputStream(EMPTY_BYTE_ARRAY));
-    // retries are not safe when the input is an InputStream, so we can't retry.
-    return Blob.fromPb(this, storageRpc.create(blobPb, inputStreamParam, optionsMap));
+  public Blob create(BlobInfo blobInfo, InputStream content, BlobTargetOption... options)
+      throws IOException {
```

Comment:
This also should be in storage PR. I don't think changing implementation just to remove `@Deprecated` is a good idea.